lib/hexpm_web/controllers/api/short_urls_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule HexpmWeb.API.ShortURLsController do
```

Comment:
controllers have singular names:

```suggestion
defmodule HexpmWeb.API.ShortURLController do
```